atmel-samd/main.c

```diff
@@ -193,13 +193,13 @@ void reset_samd21(void) {
 
 #ifdef EXPRESS_BOARD
     audioout_reset();
+#endif
+
     touchin_reset();
```

Comment:
Oops! We should think of a better way to manage these resets. Thanks for fixing it!